extension/src/cli/dvc/contract.ts

```diff
@@ -129,8 +129,117 @@ export const experimentHasError = (
 
 export type ExpShowOutput = (ExpState & { experiments?: ExpRange[] | null })[]
 
+export enum PlotsType {
+  VEGA = 'vega',
+  IMAGE = 'image'
+}
+export const isImagePlotOutput = (plot: {
+  type: PlotsType
+}): plot is ImagePlotOutput => plot.type === PlotsType.IMAGE
+
+export const PLOT_REV_FIELD = 'rev' as const
+export const PLOT_COLOR_ANCHOR = '<DVC_METRIC_COLOR>' as const
+export const PLOT_COLUMN_ANCHOR = '<DVC_METRIC_COLUMN>' as const
+export const PLOT_DATA_ANCHOR = '<DVC_METRIC_DATA>' as const
+export const PLOT_SHAPE_ANCHOR = '<DVC_METRIC_SHAPE>' as const
+export const PLOT_STROKE_DASH_ANCHOR = '<DVC_METRIC_STROKE_DASH>' as const
+export const PLOT_METRIC_TYPE_ANCHOR = '<DVC_METRIC_TYPE>' as const
+export const PLOT_TITLE_ANCHOR = '<DVC_METRIC_TITLE>' as const
+export const PLOT_X_ANCHOR = '<DVC_METRIC_X>' as const
+export const PLOT_X_LABEL_ANCHOR = '<DVC_METRIC_X_LABEL>' as const
+export const PLOT_Y_ANCHOR = '<DVC_METRIC_Y>' as const
+export const PLOT_Y_LABEL_ANCHOR = '<DVC_METRIC_Y_LABEL>' as const
+export const PLOT_ZOOM_AND_PAN_ANCHOR = '<DVC_METRIC_ZOOM_AND_PAN>' as const
+export const PLOT_PARAM_TYPE_ANCHOR = '<DVC_PARAM_TYPE>' as const
+export const PLOT_HEIGHT_ANCHOR = '<DVC_METRIC_PLOT_HEIGHT>' as const
+export const PLOT_WIDTH_ANCHOR = '<DVC_METRIC_PLOT_WIDTH>' as const
```

Comment:
Should we have these inside an enum / const? So we can have all tags while importing only one constant (autocompletion as well).